src/user-guide/writing-workflows/external-triggers.rst

```diff
@@ -82,8 +136,8 @@ cycle point value by one hour:
        [[foo]]
            script = run-foo.sh
 
-Notice that the short label ``clock_1`` is used to represent the
-trigger function in the graph.
+Notice that the short label ``clock_1`` represents the trigger function in the
+graph.
 
 Argument keywords can be omitted if called in the right order, so the
```

Comment:
Is this a necessary thing to say?

Reply:
I think it is necessary, to explain why you can call clock triggers the commonly-used shorthand way, but that for other xtriggers with more arguments, the order has to be right. It should be mentioned in the general section though.